cmd/istio-cni/main.go

```diff
@@ -221,12 +224,24 @@ func cmdAdd(args *skel.CmdArgs) error {
 				logrus.WithFields(logrus.Fields{
 					"ContainerID": args.ContainerID,
 					"netns":       args.Netns,
-					"pod":         string(k8sArgs.K8sPodName),
-					"Namespace":   string(k8sArgs.K8sPodNamespace),
+					"pod":         string(k8sArgs.K8S_POD_NAME),
+					"Namespace":   string(k8sArgs.K8S_POD_NAMESPACE),
 					"ports":       ports,
+					"annotations": annotations,
 				}).Infof("Updating iptables redirect for Istio proxy")
-
-				_ = setupRedirect(args.Netns, ports)
+				if val, ok := annotations[injectAnnotationKey]; ok {
+					logrus.Infof("Pod %s contains inject annotation: %s", string(k8sArgs.K8S_POD_NAME), val)
+					if b, err := strconv.ParseBool(val); err == nil {
```

Comment:
consider change 'b' to something more meaningful.

Reply:
done